src/packages/indicator/indicator.scss

```
@@ -49,44 +52,71 @@
     background-color: transparent;
+    border: none;
 
     &-active {
+      position: relative;
+      z-index: 1;
       transition: transform 0.3s ease-in-out;
       background: $indicator-color;
+      border: $indicator-dot-border;
+      transform: translateX(
+        calc(
+          var(--nutui-indicator-current, 0) /
```

Stylelint (17.12.0)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)